tests/integrations/aws_lambda/test_aws.py

```diff
@@ -99,13 +118,19 @@ def inner(code, payload, timeout=30, syntax_check=True):
             line = line[len(b"EVENT: ") :]
             events.append(json.loads(line.decode("utf-8")))
 
-        return events, response
+        for line in base64.b64decode(response["LogResult"]).splitlines():
```

Comment:
I would merge this with the previous loop such that events and envelopes are parsed in one go

Reply:
Parsed it in one go. Change will be in next commit.